pkgs/top-level/rust-packages.nix

```diff
@@ -21,7 +22,7 @@ let
 
 in
 
-runCommand "rustRegistry-${version}-${builtins.substring 0 7 rev}" { inherit src; } ''
```

Comment:
might be still useful to have `rev` here. This could be an optional string if `src` is not overwritten by the user.